WebServices/AzureNotificationHub/NotificationHubSample/NotificationHubSample/MainPage.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:NotificationHubSample"
+             x:Class="NotificationHubSample.MainPage">
+
+    <StackLayout>
```

Comment:
Will need a Margin setting on it (Margin="0,35,0,0") to avoid the notch on iPhone X.